opendj-doc-generated-ref/src/main/assembly/sdk-tools-man-pages-assembly.xml

```diff
@@ -27,11 +27,6 @@
     The .jar to has this file at /man-pages/man-addrate.xml for example.
   -->
   <files>
```

Comment:
Similar comment as in the previous case... by deleting these references we are loosing information about what we should add in the future. I would either comment these references out or replace content of the referenced files with some sort of `TODO` comment.

Reply:
I created empty man page files with CDDL license. So this change has been reverted.